docroot/themes/humsci/humsci_basic/src/scss/utilities/_wysiwyg-text-area.scss

```diff
@@ -177,67 +211,11 @@ blockquote:nth-child(n) {
     @include hb-blockquote;
 
     .hb-local-footer--dark & {
-      @include hb-colorful {
-        @include hb-global-color('color', 'gray-light');
-      }
+      @include hb-global-color('color', 'gray-light');
```

Comment:
This change didn't cause it, but I'm seeing an edge-case where non-p-tag dark-footer content isn't getting style overrides:

![image](https://user-images.githubusercontent.com/1256329/81736748-0f72a200-9465-11ea-8926-28ef52740754.png)

I wonder if now is a good time to put in a quick fix?

Reply:
@bryanbraun I think it's worth checking if this is an actual thing H&S would want to do, it does seem a bit strange to have bullets in a blockquote. If they do want this behavior, we can add something the Q&A card.